src/package.json

```diff
@@ -8,6 +8,7 @@
         "unity"
     ],
     "dependencies": {
+        "com.unity.test-framework.performance": "2.8.0-preview"
```

Comment:
@kevinmv I don't think I can take this dependency since Mathematics is a verified package and can't depend on preview packages.

This was part of the reason why I had to npmignore the performance tests so I didn't have to depend on this at the package level and instead keep the performance tests only in the test project.

Are you adding this dependency so that everything works when the package is embedded locally?

Reply:
I added it so that if someone did make mathematics part of `testables` then the test-framework.performance would be upgraded to the version math is using.

Today it no longer works with the 2.x line, and my change promotes to to be 2.8+. Unless there is a way to add this dependency for just tests, I think we have at least a few options:
- I comment out the SampleGroup changes and they can be re-enabled once test-framework.performance isn't in preview (I have no idea when that will be)
- I leave the code changes as is but remove the dependency. This means if you did include math in your `testables` list, the project would need to already be on test-framework.performance-2.8.0-preview or else we will fail to build. This is easily fix for internal/monorepo sample projects but is not ideal compared to auto-upgrades
- I remove the API breaking change and don't set the sample units. This allows for 1.x, and 2.x compatibility but measurements are then reported as ms by default instead of the original us.
-